shell/src/main/java/alluxio/cli/fsadmin/report/ConfigurationCommand.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.cli.fsadmin.report;
+
+import alluxio.client.MetaMasterClient;
+import alluxio.wire.ConfigProperty;
+
+import java.io.IOException;
+import java.io.PrintStream;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+
+/**
+ * Prints runtime configuration information.
+ */
+public class ConfigurationCommand {
+  private static final String CONFIG_INFO_FORMAT = "%-40s %-50s %-30s%n";
```

Comment:
is there a way to format so that the size of each column is based on the longest string in that column?

Reply:
I traverse the configProperty list to get the max length of name, value and source.
After that, I can have the dynamic config info format. 
Thanks!